### PR TITLE
[Snappi] Disable skip PFC watchdog tests for switches with broadcom ASICs

### PR DESCRIPTION
**Pull request by developfast**
### Description of PR

Summary: With the introduction of warm-up traffic to all PFC watchdog cases (#7712 and #7499), we no longer need to worry about PFC watch dog fake alerts being read by Broadcom ASICs, as its PFC watch dog ASIC logic is no longer relevant when there is an initial flow of traffic on the test priorities. Earlier, switches with Broadcom ASICs would skip PFC watchdog tests where the PFC watch dog was not being intentionally triggered, as it could un-intentionally trigger the PFC watch dog when not needed. 

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [x] 201911
- [x] 202012
- [x] 202205

#### What is the motivation for this PR?
To increase test coverage on the switches with Broadcom ASICs. Earlier these switches would not run PFC watch dog cases where PFC watch dog was not being triggered, but with code changes to add warm-up traffic, this issue is no longer present, and hence, we can re-enable back certain PFC watch dog cases. 

#### How did you do it?
Removed the skip_pfcwd_test function and its references to have all switches run all cases of PFC watch dog test cases. 

#### How did you verify/test it?
To be done. Waiting on an appropriate testbed.

### CODE REVIEW

**Review by neethajohn — DISMISSED** [review]

**Review by baiwei0427 — APPROVED** [review]

**Comment by developfast** [comment]
DO NOT MERGE UNTIL #7712 IS MERGED.

**Review by bingwang-ms — APPROVED** [review]

**Review by baiwei0427 — APPROVED** [review]

**Pull request merged**
